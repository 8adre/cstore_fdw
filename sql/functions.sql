--
-- Test utility functions for cstore_fdw tables.
--

CREATE FOREIGN TABLE empty_table (a int) SERVER cstore_server;
CREATE FOREIGN TABLE table_with_data (a int) SERVER cstore_server;

COPY table_with_data FROM STDIN;
1
2
3
\.

SELECT cstore_table_size('empty_table') < cstore_table_size('table_with_data');

DROP FOREIGN TABLE empty_table;
DROP FOREIGN TABLE table_with_data;
