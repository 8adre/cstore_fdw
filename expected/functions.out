--
-- Test utility functions for cstore_fdw tables.
--
CREATE FOREIGN TABLE empty_table (a int) SERVER cstore_server;
CREATE FOREIGN TABLE table_with_data (a int) SERVER cstore_server;
COPY table_with_data FROM STDIN;
SELECT cstore_table_size('empty_table') < cstore_table_size('table_with_data');
 ?column? 
----------
 t
(1 row)

DROP FOREIGN TABLE empty_table;
DROP FOREIGN TABLE table_with_data;
